SignalMessaging/environment/NotificationSounds.h

```diff
@@ -1,37 +0,0 @@
-//
-//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
-//
-
-typedef NS_ENUM(NSUInteger, NotificationSound) {
-    NotificationSound_Aurora = 0,
-    NotificationSound_Bamboo,
```

Comment:
Renamed to `OWSSounds` and generalized to handle notification sounds, ringtones and "call sounds".